examples/read_workplaces_with_industry_naics.py

```diff
@@ -1,8 +1,5 @@
-import numpy as np
-import sciris as sc
 import synthpops as sp
 
-
 if __name__ == '__main__':
     n = int(50e3)
```

Comment:
Highly suggest n is smaller for an example so it doesn't take as long to run.

Reply:
Ooh, looks like the data dir only has the 
>seattle_metro_10000_synthetic_workplaces_by_industry_with_uids.dat
file in the 50k version.  Could we add a 10k version?